### PR TITLE
Cleanup translated texts and update the POT file

### PR DESCRIPTION
**Pull request by DRracer**
This PR encapsulates all the work necessary to finalize texts and translations for releasing FW 4.3-RC1.
Each commit adds a standalone piece of improvement and is documented accordingly.

BFW-1434
BFW-1418
BFW-1318
BFW-1239

### CODE REVIEW

File: src/lang/po/Prusa-Firmware-Buddy.pot [hunk 1]
```diff
@@ -30,271 +30,233 @@ msgid ""
...
 #. c=20 r=6
-#: src/gui/wizard/screen_wizard.cpp:406
+#: src/gui/wizard/firstlay.cpp:130
 #, c-format
 msgid ""
-"Do you want to use\n"
-"the current value?\n"
-"Current: %0.3f.   \n"
-"Default: %0.3f.   \n"
+"Do you want to use the current value? Current: %0.3f.\n"
```
Comment (espr14) [inline src/lang/po/Prusa-Firmware-Buddy.pot:197]:
```suggestion
"Do you want to use the current value?\nCurrent: %0.3f.\n"
```

File: src/lang/po/Prusa-Firmware-Buddy.pot [hunk 2]
```diff
@@ -305,264 +267,255 @@ msgstr ""
...
+#. ***************************************************************************
+#. clang-format off
+#: src/gui/dialogs/DialogG162.cpp:8 src/gui/dialogs/DialogLoadUnload.cpp:14
+msgid "Finishing buffered gcodes."
 msgstr ""
 
 #. r=1 c=20
-#: src/gui/screen_menu_version_info.cpp:64
+#: src/gui/screen_menu_version_info.cpp:60
 msgid "Firmware Version\n"
```
Comment (espr14) [inline src/lang/po/Prusa-Firmware-Buddy.pot:292]:
```suggestion
msgid "Firmware Version"
```

File: src/lang/po/Prusa-Firmware-Buddy.pot [hunk 3]
```diff
@@ -305,264 +267,255 @@ msgstr ""
...
 
-#: src/gui/screen_print_preview.cpp:147 src/gui/screen_print_preview.cpp:155
+#: src/gui/screen_print_preview.cpp:128
 msgid "Material"
 msgstr ""
 
-#: src/gui/wizard/xyzcalib.cpp:78
+#: src/gui/wizard/xyzcalib.cpp:196
 msgid ""
 "Measuring reference\n"
```
Comment (espr14) [inline src/lang/po/Prusa-Firmware-Buddy.pot:496]:
```suggestion
"Measuring reference height of calibration points"
```

File: src/lang/po/Prusa-Firmware-Buddy.pot [hunk 4]
```diff
@@ -699,45 +620,46 @@ msgstr ""
...
 
-#: src/gui/screen_menu_lan_settings.cpp:290
+#: src/gui/screen_menu_lan_settings.cpp:280
 msgid "Please insert a USB drive and try again."
 msgstr ""
 
-#: src/gui/guimain.cpp:197
+#: src/gui/guimain.cpp:187
 msgid ""
 "Please insert the USB\n"
```
Comment (espr14) [inline src/lang/po/Prusa-Firmware-Buddy.pot:664]:
```suggestion
"Please insert the USB drive that came with the firmware"
```

File: src/lang/po/Prusa-Firmware-Buddy.pot [hunk 5]
```diff
@@ -746,35 +668,35 @@ msgid ""
...
-"push filament into\n"
-"the extruder.     "
+#: src/gui/dialogs/DialogLoadUnload.cpp:23
+msgid "Press CONTINUE and push filament into the extruder."
 msgstr ""
 
-#: src/gui/wizard/screen_wizard.cpp:205
+#: src/gui/wizard/screen_wizard.cpp:49
 msgid ""
 "Press NEXT to run  \n"
```
Comment (espr14) [inline src/lang/po/Prusa-Firmware-Buddy.pot:701]:
```suggestion
"Press NEXT to run  the Selftest, which checks for potential issues related to the assembly."
```

File: src/lang/po/Prusa-Firmware-Buddy.pot [hunk 6]
```diff
@@ -784,275 +706,307 @@ msgid ""
...
 #. r=1 c=34
-#: src/common/bsod.cpp:259
+#: src/common/bsod.cpp:257
 msgid "Scan me for details"
 msgstr ""
 
-#: src/gui/wizard/xyzcalib.cpp:66 src/gui/wizard/xyzcalib.cpp:68
+#: src/gui/wizard/xyzcalib.cpp:184
 msgid ""
 "Searching bed\n"
```
Comment (espr14) [inline src/lang/po/Prusa-Firmware-Buddy.pot:856]:
```suggestion
"Searching bed calibration points"
```

File: src/lang/po/Prusa-Firmware-Buddy.pot [hunk 7]
```diff
@@ -784,275 +706,307 @@ msgid ""
...
-#: src/gui/screen_menu_lan_settings.cpp:293
+#: src/gui/screen_menu_lan_settings.cpp:283
 msgid ""
 "The settings have been saved successfully in the \"lan_settings.ini\" file."
 msgstr ""
 
-#: src/gui/wizard/screen_wizard.cpp:191
+#: src/gui/wizard/screen_wizard.cpp:35
 msgid ""
 "The status bar is at\n"
```
Comment (espr14) [inline src/lang/po/Prusa-Firmware-Buddy.pot:1011]:
```suggestion
"The status bar is at the bottom of the screen. It contains information about:\n"
                                  " - Nozzle temp.\n"
                                  " - Heatbed temp.\n"
                                  " - Printing speed\n"
                                  " - Z-axis height\n"
                                  " - Selected filament"
```

File: src/lang/po/Prusa-Firmware-Buddy.pot [hunk 8]
```diff
@@ -1065,7 +1019,18 @@ msgid ""
 " - Selected filament"
 msgstr ""
 
-#: src/gui/screen_menu_lan_settings.cpp:296
+#: src/gui/wizard/screen_wizard.cpp:304
+msgid ""
+"The status bar is at the bottom of the screen. It contains information "
```
Comment (espr14) [inline src/lang/po/Prusa-Firmware-Buddy.pot:1024]: duplicit

File: src/lang/po/Prusa-Firmware-Buddy.pot [hunk 9]
```diff
@@ -18,7 +18,7 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 
 #. c=20 r=4
-#: src/gui/screen_menu_version_info.cpp:86
+#: src/gui/screen_menu_version_info.cpp:82
 #, c-format
 msgid ""
 "\n"
```
Comment (espr14) [inline src/lang/po/Prusa-Firmware-Buddy.pot:24]: msgid "Bootloader Version"
msgid "Buddy Board"

Comment (DRracer) [reply src/lang/po/Prusa-Firmware-Buddy.pot:24]: No way! Here the '\n' are important

File: src/lang/po/Prusa-Firmware-Buddy.pot (hunk 2 above)
Comment (DRracer) [reply src/lang/po/Prusa-Firmware-Buddy.pot:292]: No way, unless you fix the version info screen

File: src/lang/po/Prusa-Firmware-Buddy.pot (hunk 3 above)
Comment (DRracer) [reply src/lang/po/Prusa-Firmware-Buddy.pot:496]: Have you seen the actual code? It is #if0 - will be fixed later. I'm not touching this.

File: src/lang/po/Prusa-Firmware-Buddy.pot (hunk 4 above)
Comment (DRracer) [reply src/lang/po/Prusa-Firmware-Buddy.pot:664]: Again - have you seen the actual code? This is a text from an early welcome screen. I'm not touching this.

File: src/lang/po/Prusa-Firmware-Buddy.pot (hunk 5 above)
Comment (DRracer) [reply src/lang/po/Prusa-Firmware-Buddy.pot:701]: Again, #if 0 ... I won't change this

File: src/lang/po/Prusa-Firmware-Buddy.pot (hunk 6 above)
Comment (DRracer) [reply src/lang/po/Prusa-Firmware-Buddy.pot:856]: dtto. #if 0

File: src/lang/po/Prusa-Firmware-Buddy.pot (hunk 7 above)
Comment (DRracer) [reply src/lang/po/Prusa-Firmware-Buddy.pot:1011]: dtto. #if 0

File: src/lang/po/Prusa-Firmware-Buddy.pot (hunk 8 above)
Comment (DRracer) [reply src/lang/po/Prusa-Firmware-Buddy.pot:1024]: When the #if 0 code goes away, the duplicity will be removed. No change for now.

**Review by espr14 — APPROVED** [review]

**Pull request merged**